### PR TITLE
install-cni.sh tweaks for performance

### PR DESCRIPTION
**Pull request by jingyuanliang**
* Include `nice` command allowing setting lower nice values
* Output all log entries with timestamps for easier observation
* Make ENABLE_BANDWIDTH_PLUGIN optional preparing for removal
* Remove unnecessary `cat` usage and avoid `jq -c` solely for log prettiness

/assign @marqc

### CODE REVIEW

File: scripts/install-cni.sh [hunk 1]
```diff
@@ -209,14 +211,14 @@ function fillSubnetsInCniSpecV2Template {
...
+      log "IPv6 subnet detected in .spec.podCIDRs: '${subnet:-}'"
       POPULATE_IP6TABLES="true"
-      echo "ip6tables will be populated because IPv6 podCIDR is configured (from .spec.podCIDRs)"
+      log "ip6tables will be populated because IPv6 podCIDR is configured (from .spec.podCIDRs)"
       ipv6_subnet_configured="true"
       SUBNETS_REPLACEMENT+=("$(jq -nc --arg subnet "${subnet}" '[{"subnet": $subnet}]')")
       ROUTES_REPLACEMENT+=('{"dst": "::/0"}')
     else
-      echo "[ERROR] Subnet detected in .spec.podCIDRs '${subnet}' is not a valid IP range"
+      log "[ERROR] Subnet detected in .spec.podCIDRs '${subnet}' is not a valid IP range"
```
Comment (marqc) [inline scripts/install-cni.sh:221]: Maybe it's better to introduce log level on the log handler method instead. I.e. separate methods for log_debug, log_info and log_error. Also log_error might have built-in script termination with error code.

Comment (jingyuanliang) [reply scripts/install-cni.sh:221]: The current logs are pretty random... I just replaced all echo with log for the purpose of appending timestamps. Maybe more improvements later if we have capacity.

Comment (marqc) [reply scripts/install-cni.sh:221]: Sounds good! It's definitely an improvement even in current state.

File: scripts/install-cni.sh [hunk 2]
```diff
@@ -14,21 +14,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+log() {
+  echo "[${EPOCHREALTIME}]" "$@"
```
Comment (sypakine) [inline scripts/install-cni.sh:18]: Users will ingest these logs via stackdriver or other logging system -- can we rely on those to provide the timestamps from the container logs?

Comment (jingyuanliang) [reply scripts/install-cni.sh:18]: I made them for easier observation from direct `kubectl logs` calls. Things like kubelet logs also have timestamps.

Comment (jingyuanliang) [reply scripts/install-cni.sh:18]: If it's a `$(date)` call I wouldn't do it, but `${EPOCHREALTIME}` being bash builtin I assume the overhead is negligible?

Comment (jingyuanliang) [reply scripts/install-cni.sh:18]: Also in case of ingestion delay - we're trying to optimize for sub-second delay here.

Comment (sypakine) [reply scripts/install-cni.sh:18]: The container logs already have timestamps:

```
$ kubectl logs example --timestamps
2024-05-21T18:44:08.265571760Z Hello, World!
```

This is the same timestamp provided by stackdriver in the `timestamp` field (and is distinct from `receivedTimestamp`).

Comment (jingyuanliang) [reply scripts/install-cni.sh]: Wow didn't know about this, but let's leave the `log` call for further extension (e.g. some messages to be written to stderr).

Comment (sypakine) [reply scripts/install-cni.sh]: SGTM

**Pull request merged**